ci/do_ci.sh

```diff
@@ -96,14 +107,11 @@ if [ -n "$CIRCLECI" ]; then
         mv "${HOME:-/root}/.gitconfig" "${HOME:-/root}/.gitconfig_save"
         echo 1
     fi
-
-    NUM_CPUS=8
-    if [ "$1" == "coverage" ]; then
-        NUM_CPUS=6
-    fi
+    # We constrain parallelism in CI to avoid running out of memory.
+    NUM_CPUS=6
```

Comment:
We shouldn't need to do this if we switch to Clang I think.

Reply:
removed